apps/judicial-system/web/src/components/InfoCard/DefendantInfo/DefendantInfo.tsx

```diff
@@ -75,19 +80,16 @@ export const DefendantInfo: FC<PropsWithChildren<DefendantInfoProps>> = (
           </Text>
         </span>
 
-        <div>
-          <Text as="span">
-            {getAppealExpirationInfo(defendant.verdictViewDate ?? '')}
-          </Text>
-        </div>
+        {displayAppealExpirationInfo && (
+          <div>
```

Comment:
Consider using `Box` with a `component="span"` prop. That saves us from having to add a div..